gluco-check-frontend/src/components/Banner.tsx

```diff
@@ -0,0 +1,97 @@
+import { useTranslation } from "react-i18next";
+import {
+  Avatar,
+  Box,
+  Button,
+  Container,
+  Divider,
+  Fab,
+  Grid,
+  makeStyles,
+  Paper,
+  Typography,
+} from "@material-ui/core";
+import { Warning } from "@material-ui/icons";
+import { JOIN_BETA } from "../lib/constants";
+
+const useStyles = makeStyles((theme) => ({
+  title: {
+    fontSize: theme.typography.body1.fontSize,
+    fontWeight: 600,
+    color: theme.palette.text.secondary,
+  },
+  subtitle: {
+    fontSize: theme.typography.body2.fontSize,
+  },
+  icon: {
+    backgroundColor: theme.palette.warning.main,
+    color: theme.palette.warning.contrastText,
+    marginTop: theme.spacing(1),
+  },
+  button: {
+    fontWeight: 600,
+    backgroundColor: theme.palette.warning.main,
+    color: theme.palette.warning.contrastText,
+    "&:hover": {
+      backgroundColor: theme.palette.warning.dark,
+    },
+  },
+}));
+
+function Banner() {
+  const { t, i18n } = useTranslation();
+
+  // TODO: can't we just get this straight from i18n?
```

Comment:
I don't think we can. i18next has no concept of the languages it's translating to. It just takes in a language code like 'en' uses it to substitute placeholders with the strings in the corresponding json file